scripts/zkai_attention_kv_ezkl_same_surface_export_probe.py

```
@@ -0,0 +1,484 @@
+    required_commitments = (
+        "input_steps_commitment",
+        "outputs_commitment",
+        "public_instance_commitment",
+        "score_row_commitment",
+        "statement_commitment",
+        "weight_table_commitment",
+    )
+    for key in required_commitments:
+        value = source.get(key)
+        if not isinstance(value, str) or not value.startswith("blake2b-256:"):
+            raise SameSurfaceExportProbeError(f"missing or invalid commitment field {key}")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)